### PR TITLE
Feature/11903 module calendrier

### PR DESCRIPTION
**Pull request by erochOnGit**

### CODE REVIEW

File: src/web/theme/pages/TryIt/SelectDate/SelectDate.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <div className="select-date__calendar">
+          <DayPickerSingleDateController
+            date={date}
+            onDateChange={date => {
+              setStepIsFilled(true);
+              setDate(date);
+              setTryItState({ date: date._d });
+            }}
+            focused={true}
+            onFocusChange={({ focused }) => setFocused(true)}
```
Comment (milkae) [inline src/web/theme/pages/TryIt/SelectDate/SelectDate.js]: What does focused is used for ?

File: src/web/theme/pages/TryIt/SelectStore/SelectStore.js [hunk 2]
```diff
@@ -24,83 +26,75 @@ const SelectStore = ({
...
+        stepIndex={getStepIndex(stepName)}
+        actionDescription="Select a store location on the map"
+        collapsed={collapsed}
+      >
+        <div className="select-store__searchbar">
+          <Select
+            value={
+              loading
+                ? null
+                : addresses.filter(selectAddress => {
```
Comment (milkae) [inline src/web/theme/pages/TryIt/SelectStore/SelectStore.js]: Seems that you can directly use `address` as value

File: src/web/theme/pages/TryIt/Step/Step.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    font-weight: 700;
+    display: flex;
+    align-items: baseline;
+
+    & > * {
+      margin-right: 1em;
+    }
+
+    &__pins,
+    #{$this}--validated &__pins {
```
Comment (hihuz) [inline src/web/theme/pages/TryIt/Step/Step.scss]: I know that I suggested to try something like this but when looking at it now it doesn't seem any clearer than repeating the full selector :laughing:

File: src/web/theme/pages/TryIt/SelectStore/SelectStore.js (hunk 2 above)
Comment (erochOnGit) [reply src/web/theme/pages/TryIt/SelectStore/SelectStore.js]: nope.

File: src/web/theme/pages/TryIt/Step/Step.scss (hunk 3 above)
Comment (erochOnGit) [reply src/web/theme/pages/TryIt/Step/Step.scss]: Maybe I wrote it the wrong way ?

**Pull request merged**